source/type/pb_converter.h

```diff
@@ -34,6 +34,108 @@ www.navitia.io
 #include "type/response.pb.h"
 #include "type/pt_data.h"
 
+namespace pt = boost::posix_time;
+
+// we don't use a boolean to have a more type checks
+enum class DumpMessage {
+    Yes,
+    No
+};
```

Comment:
eventually, we can use a real boolean for DumpMessage now that it is in an object.